deploy/kubernetes/elasticsearch/elasticsearch.gcloud.yaml

```diff
@@ -1,63 +1,183 @@
+    podTemplate:
+      metadata:
+        labels:
+          name: elasticsearch
+          deployment: {{ DEPLOY_TO }}
+      spec:
+        affinity:
+          nodeAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
+              nodeSelectorTerms:
+              - matchExpressions:
+                - key: nodeType
+                  operator: NotIn
+                  values:
+                  - data
+        initContainers:
+          - name: sysctl
+            securityContext:
+              privileged: true
+            command: ['sh', '-c', 'sysctl -w vm.max_map_count=262144']
+        containers:
+        - name: elasticsearch
+          imagePullPolicy: {{ IMAGE_PULL_POLICY }}
+  - name: data
+    count:  {{ ES_DATA_NUM_PODS }}
+    config:
+      node.master: false
+      node.data: true
+      node.ingest: false
+      node.ml: false
+      node.remote_cluster_client: false
+    podTemplate:
+      metadata:
+        labels:
+          name: elasticsearch
+          deployment: {{ DEPLOY_TO }}
+      spec:
+        affinity:
+          nodeAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
+              nodeSelectorTerms:
+              - matchExpressions:
+                - key: nodeType
+                  operator: In
+                  values:
+                  - data
+        initContainers:
+        - name: sysctl
+          securityContext:
+            privileged: true
+          command: ['sh', '-c', 'sysctl -w vm.max_map_count=262144']
+        containers:
+        - name: elasticsearch
+          imagePullPolicy: {{ IMAGE_PULL_POLICY }}
+          env:
+          - name: ES_JAVA_OPTS
+            value: "-Xms{{ ELASTICSEARCH_JVM_MEMORY }} -Xmx{{ ELASTICSEARCH_JVM_MEMORY }}"
+          resources:
+            requests:
+              memory: {{ ELASTICSEARCH_CONTAINER_MEMORY }}
+            limits:
+              memory: {{ ELASTICSEARCH_CONTAINER_MEMORY }}
+# for use in mounting multiple disks to copy data - should not be used for real in production
+#          volumeMounts:
+#          - name: storage
+#            mountPath: /data
+#        volumes:
+#        - name: storage
+    volumeClaimTemplates:
+    - metadata:
+        name: elasticsearch-data
+      spec:
+        accessModes:
+          - ReadWriteOnce
+        storageClassName: {{ ES_DATA_DISK_TYPE }}-storage-class
         resources:
           requests:
-            memory: "2Gi"
-            cpu: "0.01"
-          limits:
-            memory: "15Gi"
-            cpu: "2"
-        volumeMounts:
-        - name: elasticsearch-dbpath
-          mountPath: /elasticsearch-data
-        env:
-        - name: ELASTICSEARCH_SERVICE_PORT
-          value: "{{ ELASTICSEARCH_SERVICE_PORT }}"
-      restartPolicy: Always
-      dnsPolicy: ClusterFirst
+            storage: {{ ES_DATA_DISK_SIZE }}
+# for use in mounting multiple disks to copy data - should not be used for real in production
```

Comment:
When I create the cluster initially, I need to do a one time rearrangement of files on the disks, which require me to mount a second blank disk/ See comment on the ticket for a better explanation: https://github.com/macarthur-lab/seqr-private/issues/865#issuecomment-678440151